src/systems/TrainSecuritySystem.cpp

```diff
@@ -210,13 +209,13 @@ namespace godot {
         mover->SecuritySystem.MaxHoldTime = ca_max_hold_time;
 
         switch (emergency_brake_warning_signal) {
-            case EmergencyBrakeWarningSignal::SIREN_LOW_TONE:
+            case SIREN_LOW_TONE:
```

Comment:
moze nie trzeba prefiksować, ale lepiej było widać z jakiego enuma to pochodzi. teraz - nie widać. (readability counts)

Reply:
Ta ale wiadomo ze eunym pochdoza z tej klasy.
Jesliby 2 enumy mialy te same wartosci - trzeba, badz jakby byly z innych klas
Dlugie nazwy obiektow/enumow w linijkach tez moga czesto zaszkodzic czytelnosci dlatego jak sie da to sie ich pozbywam ;)

Reply:
Czy wiadomo jaki to konkretnie enum? Bo ja tego nie widze mimo wszystko. Moze byc w jednej klasie kilka deklaracji. Sama klasa moze sie rozrosnac.

Ps. Moze kiedys popracujemy nad coding standards i to spiszemy (czesciowo robi to clang-format)? Chodzi o to zeby kodowac w odpowiednim stylu, i zeby na review w kolko o tym nie gadac.

Reply:
Nie można mieć dwóch rzeczy o takich samych nazwach w klasie więc wiadomo jaki to enum i z jakiej klasy nawet bez deklaracji :)

Albo można jakby nazwa była krótsza :)

Reply:
Jeszcze jest enum class, ktory osobiscie wole, ale godot go nie lubi. 

Jesli to jest global namespace, to wartosci enumow powinismy wszedzie prefixowac, czyli wlasciwie mozna byloby zostac przy #define